native/src/modules/endpoint/hocs/withPayloadProvider.js

```diff
@@ -95,33 +95,28 @@ const withPayloadProvider = <S: { dispatch: Dispatch<StoreActionType> }, R: {}>
       if (props.status === 'routeNotInitialized') {
         return null
       } else if (props.status === 'error') {
-        return <ScrollView refreshControl={<RefreshControl onRefresh={refreshIfPossible} refreshing={false} />}
-                           contentContainerStyle={{ flexGrow: 1 }}>
+        return <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
```

Comment:
This should be reverted to keep the pull to refresh behavior. As long as `refreshing={false}` no spinner is shown.

Reply:
done